pkger/clone_resource.go

```diff
@@ -832,11 +832,14 @@ func convertCellView(cell influxdb.Cell) chart {
 		ch.Kind = chartKindMosaic
 		ch.Queries = convertQueries(p.Queries)
 		ch.Colors = stringsToColors(p.ViewColors)
+		ch.HoverDimension = p.HoverDimension
 		ch.XCol = p.XColumn
 		ch.GenerateXAxisTicks = p.GenerateXAxisTicks
 		ch.XTotalTicks = p.XTotalTicks
 		ch.XTickStart = p.XTickStart
 		ch.XTickStep = p.XTickStep
+		ch.YLabelColumnSeparator = p.YLabelColumnSeparator
```

Comment:
Should we do anything special here if `YLabelColumnSeparator` is empty, for backwards-compatibility?

Reply:
No need. Empty string will be interpreted correctly by the UI.